docs/types/type-inference.md

```diff
@@ -125,13 +125,13 @@ We have already seen how [Type Guards](./typeGuard.md) help change and narrow do
 
 ### Be careful around parameters
 
-Types do not flow into the function parameters if it cannot be inferred from an assignment. e.g. in the following case the compiler does not to know the type of `foo` so it cannot infer the type of `a` or `b`
+Types do not flow into the function parameters if it cannot be inferred from an assignment. For example in the following case the compiler does not know the type of `foo` so it cannot infer the type of `a` or `b`.
 
 ```t
 const foo = (a,b) => { /* do something */ };
 ```
 
-However if `foo` was typed the function parameters type can be inferred (`a`,`b` are both inferred to be number below).
+However if `foo` was typed the function parameters type can be inferred (`a`,`b` are both inferred to be of type number in the example below).
```

Comment:
> of type `number`

?

Reply:
@j-f1 thanks for reviewing. Its okay to be more verbose so merged 🤷‍♀️  :rose: